hack/prepare-envtest.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ENVTEST_K8S_VERSION=${ENVTEST_K8S_VERSION:-"1.30"}
+
+echo "> Installing envtest tools@${ENVTEST_K8S_VERSION} with setup-envtest if necessary"
+if ! command -v setup-envtest &> /dev/null ; then
+  >&2 cat <<EOF
+  setup-envtest not available
+  You can either directly install it by following the instructions at https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest#section-readme
+  Or you can use `make test-integration` which will implicitly install it for you.
+EOF
```

Comment:
When the `setup-envtest` command is not available, the script mistakenly proceeds to execute the block below, leading to `make test-integration` being unintentionally triggered. 


To fix this, I recommend escaping the backticks for `make test-integration` in the output message, so it doesn't inadvertently get executed.


```diff
- Or you can use `make test-integration` which will implicitly install it for you.
+ Or you can use \`make test-integration\`, which will implicitly install it for you.
```

Reply:
Alternatively, you could remove this check, as `prepare-env.sh` is invoked by `start-env.sh`, which in turn is triggered by the `make setup-envtest` target. This ensures that the `setup-envtest` command will be installed automatically when running the `make setup-envtest` target.